lib/Cake/View/Helper.php

```diff
@@ -552,7 +552,7 @@ protected function _formatAttribute($key, $value, $escape = true) {
  */
 	protected function _confirm($message, $okCode, $cancelCode = '', $options = array()) {
 		$message = json_encode($message);
-		$confirm = "if (confirm({$message})) { {$okCode} } {$cancelCode}";
+		$confirm = "if (confirm({$message})) { {$okCode} } else { {$cancelCode} }";
```

Comment:
Fair point. The existing behavior is what people are relying on, not the documentation. I'm reluctant to change the behavior to match the documentation this far into the life of 2.x